taichi/ir/type.cpp

```diff
@@ -64,6 +64,22 @@ DataType DataType::ptr_removed() const {
   }
 }
 
+std::vector<int> DataType::get_element_shape() const {
```

Comment:
`element_shape` means shape of one's element, which is for `ndarray`. A `TensorType` should only have `shape` instead of `element_shape`.

Reply:
done. thx